docs/language-guide.rst

```diff
@@ -85,19 +87,25 @@ To define such keyword groups the attribute ``keywords`` becomes an object each
     }
   }
 
-The group name becomes then a class name in a generated markup enabling different styling for different kinds of keywords.
+The group name becomes the class name in the generated markup enabling different
+themeing for different kinds of keywords.
 
-To detect keywords highlight.js breaks the processed chunk of code into separate words — a process called lexing.
-The "word" here is defined by the regexp ``[a-zA-Z][a-zA-Z0-9_]*`` that works for keywords in most languages.
-Different lexing rules can be defined by the ``lexemes`` attribute:
+To detect keywords highlight.js breaks the processed chunk of code into separate
+words — a process called lexing. By default "words" are matched with the regexp
+``\w+``, and that works well for many languages. Different lexing rules can be
+defined by the magic ``$pattern`` attribute:
 
 ::
 
   {
-    lexemes: '-[a-z]+',
-    keywords: '-import -export'
+    keywords: {
+      $pattern: /-[a-z]+/,        // allow keywords with dash in them
```

Comment:
with dash in them -> that start with a dash?

Reply:
I was upgrading not bug fixing. :-)

Reply:
And I don't really know for sure so I left it alone.

Reply:
That's right, but I'm talking about the comment you've added (:  It is misleading.

Reply:
Oh yes you're right! I will fix that.

Reply:
Oh because it's the docs and I was trying to be helpful, I'l lcorect it to be more specific.